### PR TITLE
perf: improve VersionSelect performance

### PR DESCRIPTION
**Pull request by codebytere**
Closes https://github.com/electron/fiddle/issues/1112.

Virtualizes list rendering, thereby improving perf ~10x for large lists.

### CODE REVIEW

**Review by BlackHole1 — APPROVED** [review]
Wow. So cool!

**Comment by BlackHole1** [comment]
Because of the changes to the `yarn.lock` file, it is best not to merge this PR for now. This is because it can lead to conflicts at https://github.com/electron/fiddle/pull/1109 that are difficult to resolve.
So it's best to wait for https://github.com/electron/fiddle/pull/1109 to merge first

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "@octokit/rest": "^16.43.1",
     "@sentry/electron": "^2.5.3",
     "@types/getos": "^3.0.1",
+    "@types/react-window": "^1.8.5",
```
Comment (BlackHole1) [inline package.json:46]: Move to `devDependencies`

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Comment by codebytere** [comment]
@BlackHole1 that PR is still in a draft state and the lockfile changes in this PR are very minimal - i'm not quite sure i understand why we would pend this smaller change on that much bigger one given it's quite far from being in a mergeable state?

**Comment by dsanders11** [comment]
Agreed with @codebytere here. #1109 is still draft and may take a little while to land, and once finalized, due to the large `yarn.lock` changes I plan to do them as a separate commit I'll push to that branch anyway since we're usually weary about large, hard to review `yarn.lock` changes. So I wouldn't worry about merge conflicts on `yarn.lock` with #1109.

**Comment by BlackHole1** [comment]
@codebytere @dsanders11 Okey.

**Review by BlackHole1 — APPROVED** [review]

**Pull request merged**